schemas/mw-article-edit.js

```diff
@@ -0,0 +1,68 @@
+{
+    "title": "MediaWiki Article Edit",
+    "description": "Represents a MW Article Edit event",
+    "type": "object",
+    "properties": {
```

Comment:
> In the case of anon edits, it will contain the user's IP 

Yes and so does the public interface.  This is part of standard MediaWiki behavior. 

> having user info in these events can come in quite handy.

It is essential -- just just handy.